scripts/release-ckeditor5.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+# For licensing, see LICENSE.md.
+
+set -e
+
+read -p "Did you update the README.md? " -n 1 -r
+echo ""
+
+if [[ $REPLY =~ ^[Yy]$ ]]
+then
+	# Release the CKEditor5 repository.
+	node ./scripts/release/release-ckeditor5.js
+
+    # Update the `stable` branch in the ckeditor5 repository.
+	git checkout stable && git merge master && git checkout master
+
+	# Update all `stable` branches in all packages.
+	mgit exec 'git checkout stable && git merge master && git checkout master'
+
+	# Make sure that `mgit.json` for `stable` and `master` branches is correct.
+	# `stable` branch.
+	git checkout stable && \
+	node -e "require( './scripts/release/update-mgit-branches' )( 'stable' );" && \
```

Comment:
Why not passing `stable/master` through `args` like to any shell command?

Reply:
Mgit does not allow changing branches in `mgit.json` file via CLI. We can save hashes or check out the branches in the repo but there is no option to save particular branch in the file.

Reply:
I meant calling this:

```
./scripts/release/update-mgit-branches stable
```

Wouldn't it be easier?